package.json

```diff
@@ -49,9 +61,9 @@
   },
   "dependencies": {
     "@babel/preset-env": "^7.12.11",
+    "@ro0gr/ceibo": "^2.2.0",
```

Comment:
switched to use this one for now, until https://github.com/san650/ceibo/pull/23 is ready